src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -659,7 +651,7 @@ def __call__(
         if dropout_rng is not None:
             rngs["dropout"] = dropout_rng
 
-        pooled, logits = self.module.apply(
+        logits = self.module.apply(
```

Comment:
Same here, `pooled` is required in `run_mlm_flax.py`

Reply:
removed `pooled` from `run_mlm_flax.py` as its not used for training anyways -> think it should be fine.

Reply:
We will need the ModelOutput in another PR :-)